src/visitor/postgres.rs

```diff
@@ -365,12 +365,12 @@ impl<'a> Visitor<'a> for Postgres<'a> {
     #[cfg(feature = "postgresql")]
     fn visit_text_search(&mut self, text_search: crate::prelude::TextSearch<'a>) -> visitor::Result {
         let len = text_search.exprs.len();
-        self.surround_with("to_tsvector(", ")", |s| {
+        self.surround_with("to_tsvector(concat_ws(' ', ", "))", |s| {
             for (i, expr) in text_search.exprs.into_iter().enumerate() {
                 s.visit_expression(expr)?;
 
                 if i < (len - 1) {
-                    s.write("|| ' ' ||")?;
+                    s.write(",")?;
```

Comment:
This unrelated change in here looks scary.

Reply:
Its not unrelated. This is related to switching to using `concat_ws`

Reply:
How I understand this: the generated query fragment went from `to_tsvector('a' || ' ' || 'b' || ' ' || 'c')` (`||` is the string concatenation operator on postgres), to `to_tsvector(concat_ws(' ', 'a', 'b', 'c'))`, which is better since if any of a, b, or c is null, you will still get the other ones in the concatenated string, but not so with `||`.

Reply:
(so if `a` is NULL, `a || ' ' || 'b'` would be NULL, but `concat_ws(' ', a, 'b')` would be ` b`)

Reply:
Good explanation Tom, thanks. That is correct.

Reply:
For example, based on our tests:
Original columns:
```
postgres=# SELECT "id", "fieldA", "fieldC" from "public"."TestModel";
 id |     fieldA     | fieldC  
----+----------------+---------
  1 | Chicken Masala | 
  2 | Chicken Curry  | 
  3 | Caesar Salad   | Chicken
  4 | Beef Sandwich  | Beef
  ```
  
```
SELECT "id", "fieldA", "fieldC",  to_tsvector("fieldA" || ' ' ||  "fieldC") from "public"."TestModel";
 id |     fieldA     | fieldC  |           to_tsvector            
----+----------------+---------+----------------------------------
  1 | Chicken Masala |         | 
  2 | Chicken Curry  |         | 
  3 | Caesar Salad   | Chicken | 'caesar':1 'chicken':3 'salad':2
  4 | Beef Sandwich  | Beef    | 'beef':1,3 'sandwich':2
  ```
  
  Then using `concat_ws`
  ```
  postgres=# SELECT "id", "fieldA", "fieldC",  to_tsvector(concat("fieldA","fieldC")) from "public"."TestModel";
 id |     fieldA     | fieldC  |         to_tsvector         
----+----------------+---------+-----------------------------
  1 | Chicken Masala |         | 'chicken':1 'masala':2
  2 | Chicken Curry  |         | 'chicken':1 'curri':2
  3 | Caesar Salad   | Chicken | 'caesar':1 'saladchicken':2
  4 | Beef Sandwich  | Beef    | 'beef':1 'sandwichbeef':2
  ```

Reply:
Notice how when using `||` the first two rows are missing all text

Reply:
Looked unrelated as PR is about MySQL, change is in PostgreSQL file.